### ISSUE TITLE
I would like to know the details about fine-tune in your paper

### ISSUE THREAD

**Issue by 1184125805** [naoto0804/cross-domain-detection#42]
![image](https://user-images.githubusercontent.com/38450078/221599751-cdf801ed-0abd-4f76-affc-617ee2715893.png)
I want to know how to do fine-tune？It's just use source domain FSD as a pre-train model，and start faster rcnn again？Maybe it will lead to catastrophic forgetting according to incremental learning knowledge.So maybe I misunderstand the meaning of fine-tune,how we execute find-tune? This problem has troubled me for a long time. Thank you

**Comment by naoto0804** [comment]
Hi, thank you for your interest in our project.
I guess your understanding of fine-tuning is correct. We use a lower learning rate for fine-tuning than pre-training.

> it will lead to catastrophic forgetting

Our setting is not incremental learning; thus, the performance on the source domain does not matter. Maintaining performance on both source and target domains is another challenge, which we leave for future work.

**Comment by 1184125805** [comment]
Thank you very much for your answer, because I am a fresh man and lack of knowledge. Do you know if there is such a job which can use target domain to improve the accuracy of source domain?Have anyone done this before?
